Add inset text pointing users to v12 docs (#91)
alphagov/govuk-prototype-kit-docs: docs/v13/documentation/install/getting-started.md

There is an [getting started advanced guide](./getting-started-advanced) if you’re comfortable using Git and the terminal.

[How to run the kit](./how-to-run-the-kit)

<!-- TODO: replace with Nunjucks partial _new-version-inset-text -->
<div class="govuk-inset-text">
  <strong class="govuk-body govuk-!-font-weight-bold">

>     This has changed in version 13, if you are using an older version, see the

Reviewer: ```suggestion
    This has changed in version 13, if you are using a previous version, see the
```
